js/react_native/lib/binding.ts

```diff
@@ -64,6 +64,7 @@ export declare namespace Binding {
 
   interface InferenceSession {
     loadModel(modelPath: string, options: SessionOptions): Promise<ModelLoadInfoType>;
+    loadModelFromBase64EncodedBuffer?(buffer: string, options: SessionOptions): Promise<ModelLoadInfoType>;
     run(key: string, feeds: FeedsType, fetches: FetchesType, options: RunOptions): Promise<ReturnType>;
```

Comment:
Can we keep the base64 encoding as an implementation detail, and allow the user to provide a Uint8Array instead, maybe renaming this to loadModelFromBytes?

If we replace this binding with a JSI based solution the base64 encoding would be unnecessary.